### ISSUE TITLE
Support NeoReader for Onyx devices

### ISSUE THREAD

**Issue by sevadah** [FBarrca/obsidian-handwritten-notes#11]
Hi, 

First, thanks for your work, I have been looking for such a plugin for a long time. 
I own an Onyx eLink device with a capable PDF reader/annotate - NeoReader. It can access files in tablet storage, so I can open a PDF file from the Obsidian's vault and annotate with it, however, changes are not stored back to the file (if I open it through the annotate button from your plugin). 

I checked the settings of the NeoReader and it is set to save changes to the exact location from where it opens (indeed if I open the pdf in my Obsidian vault from the native file manager, everything works as expected and I can see annotations on the pdf). 
When I open the pdf through the plugin, and do some annotations they are not visible within the Obsidian, but when I re-open with the NeoReader they are there...
So my guess is, when I open the pdf with your plugin, you load it in some sort of temp/cache directory and then reopload it to Obsidian? 

So I wonder, what are the possible solutions, and whether/how I can help to resolve those.

Cheers

**Comment by FBarrca** [comment]
TLDR: Many apps don't work, use one of the referenced on the README. I recommend you sideload Xodo 5.0.22, which is the app that i use.

long answer:
The technical issue is that obsidian only exposes the [share api](https://capacitorjs.com/docs/apis/share) on the mobile (capacitor) versions of the app. I have never dived deep into native android apps, but it seems many apps are badly coded or are outdated and don't use the latest(from android 10) api [share recieve intent](https://developer.android.com/training/sharing/receive). Therefore, the file isn't really referenced, and instead it passes a copy to the app.
This isn't really something I can fix, as my hands are tied at both ends.
